securitas/security/ipa.py

```diff
@@ -46,6 +46,21 @@ def group_add_member_manager(
             parse_group_management_error(data)
         return data['result']
 
+    def pwpolicy_add(
+        self, group, krbminpwdlife=None, krbpwdminlength=None, cospriority=None, **kwargs
+    ):
+        """
+        Create the password time policy
```

Comment:
Actually this method can do more than create a password *time* policy, it can create the whole password policy given the proper kwargs (even if they are not explicitely listed yet). I would suggest removing `time`.